aodndata/soop/soop_ba.py

```diff
@@ -0,0 +1,119 @@
+from aodncore.pipeline import HandlerBase, PipelineFile, PipelineFilePublishType, PipelineFileCheckType, FileType
+from aodncore.pipeline import PipelineFileCollection
+from aodncore.pipeline.exceptions import InvalidInputFileError
+from aodndata.soop.soop_ba_classifier import SoopBaFileClassifier
+from aodncore.pipeline.statequery import StateQuery
+import re
+import os
```

Comment:
PEP8, these should be above. 
On  pycharm, 
Code -> optimize imports

This will remove the unused imports too